adafruit_display_shapes/roundrect.py

```diff
@@ -65,16 +55,18 @@ def __init__(self, x, y, width, height, r, *, fill=None, outline=None, stroke=1)
         self._palette = displayio.Palette(3)
         self._palette.make_transparent(0)
         self._bitmap = displayio.Bitmap(width, height, 3)
+        for i in range(0, width):   # draw the center chunk
+            for j in range(r, height - r):   # draw the center chunk
+                self._bitmap[i, j] = 2
+        self._helper(r, r, r, color=2, fill=True,
+                    x_offset=width-2*r-1, y_offset=height-2*r-1)
```

Comment:
It looks like it failed the checks because a space need to be added to this line.